components/auth/src/main/java/org/cloudfoundry/credhub/config/auth/OAuth2AuthenticationExceptionHandler.java

```diff
@@ -29,18 +29,21 @@ public void commence(HttpServletRequest request,
                          AuthenticationException authException)
             throws IOException {
         btaep.commence(request, response, authException);
-
+        if (authException == null) {
+            return;
+        }
```

Comment:
I missed this change earlier. We do not need null check here.

Reply:
Removed